src/wms/src/main/java/org/geoserver/wms/map/RenderedImageMapOutputFormat.java

```diff
@@ -173,6 +173,12 @@
     /** The lookup table used for data type transformation (it's really the identity one) */
     private static LookupTableJAI IDENTITY_TABLE = new LookupTableJAI(getTable());
 
+    /** Show Chain key */
+    public static final String RASTER_SHOWCHAIN_KEY = "wms.raster.showChain";
```

Comment:
wms.raster.enableRasterChainDebug ?

Reply:
updated accordingly. thx for the suggestion